netlify/next-runtime
# fix: follow redirects when downloading a file from the CDN

### Summary

Uses the `follow-redirects` library to make the `http` request for downloading a file from the CDN.

In cases we where needed to follow the redirect initially returned we were instead failing the request, leading to `500` errors regarding missing files as seen in #1679

### Test plan

* Visit https://ep-next-auth-investigation.netlify.app/auth/signin
* Open the network tab
* Confirm that the `signin.json` file request is returning a `200` response (might need to refresh the page
* Click 'sign in'
* The redirect should work as expected and there should be no `500` responses returned

Should be noted that the test site used above came from the [user who reported the issue](https://github.com/Joroze/next-netlify-starter-nextauth/pull/1), I just forked it.

I haven't added automated tests here because I'd be essentially testing the package itself rather than functionality we've added.

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal
Fixes #1679 

![signal-2022-10-17-21-04-17-223](https://user-images.githubusercontent.com/5655473/198107840-8012e527-1dd1-4644-836b-20ccea361706.jpg)
